docs/containers.conf.5.md

```diff
@@ -907,11 +907,24 @@ URI to access the Podman service
 - **rootless remote** - ssh://user@engineering.lab.company.com/run/user/1000/podman/podman.sock
 - **rootful local**  - unix:///run/podman/podman.sock
 - **rootful remote** - ssh://root@10.10.1.136:22/run/podman/podman.sock
+- **tcp/tls remote** - tcp://10.10.1.136:9443
 
 **identity="~/.ssh/id_rsa**
 
 Path to file containing ssh identity key
 
+**tls_cert_file="~/certs/podman/tls.crt"**
```

Comment:
I see `identity` sets the same example with `~` but AFAIK our code will never expand this as we just pass it to the open syscall and normally the shell will expand `~` so this will most likely not work.

So this example might mislead some people to thinking it works, I would recommend using full paths in the example same for all the other cases here.

Reply:
Ah, I think you're right. Do you think it'd be a good idea to require consumers (e.g. podman) to expand the tilde to be consistent with the identity field? Or does the identity example also need to be updated?

Reply:
I would just update the docs here, so far nobody has asked for it. And if you create the connections with `podman system connection add --identity ~/...`  the shell already did it for us and we just see the full path anyway so I don't think it matters to users.

It only matters for users that manually write the connections in the config file but I am not aware of anyone doing this.

Reply:
Makes sense. Done.